UnitTests/TestNetwork.cpp

```diff
@@ -968,168 +849,21 @@ TEST(TestNetwork, DISABLED_ZMQ_BIP15X_StressTest)
    }
 }
 
-TEST(TestNetwork, ZMQ_BIP15X_MalformedData)
```

Comment:
We should keep one of such malformed tests - when server and/or client send something inappropriate to the peer, and how this is handled by the counterparty.

Reply:
I did add `TestWebSocket.BrokenConnect` and `TestWebSocket.BrokenData`.